src/scripts/blocks/router/router-helper.provider.js

```diff
@@ -0,0 +1,36 @@
+function routerHelperProvider($locationProvider, $stateProvider, $urlRouterProvider) {
+  'ngInject';
```

Comment:
I see - this is necessary because we're not registering this with angular right away using provider. Maybe we should stick with the way we've been doing it? That way we won't have to have this separate way of doing ngInject, and you can also know what your function is being used as (provider, factory, etc) rather than having to trace back to where it's used.